.gitignore

```diff
@@ -71,6 +71,7 @@ gstoraster
 imagetopdf
 imagetoraster
 mupdftoraster
+mupdftopwg
```

Comment:
I have renamed `mupdftoraster` into `mupdftopwg` some weeks ago, as MuPDF (`mutool`) only outputs PWG Raster, no CUPS Raster and no Apple Raster. I simply forgot that I also had to update `.gitignore`. So could you remove `mupdftoraster` here to make it complete?

Reply:
Sure

Reply:
Thanks.